lib/unicode_plot/lookup_canvas.rb

```diff
@@ -19,17 +19,17 @@ def pixel!(pixel_x, pixel_y, color)
       pixel_y -= 1 unless pixel_y < pixel_height
 
       tx = pixel_x.fdiv(pixel_width) * width
-      char_x = tx.floor + 1
-      char_x_off = pixel_x % x_pixel_per_char + 1
-      char_x += 1 if char_x < tx.round + 1 && char_x_off == 1
+      char_x = tx.floor
```

Comment:
simlar to canvas.rb, we have 
- fixes for 1-index ==> 0-index
- fixes for rounding issue #32 
- fixes for color fetching issue #34 
-